### PR TITLE
Update SushiBeltTrackerItem properties access

### PR DESCRIPTION
**Pull request by GeekTree0101**
Making a custom visibleRatioCalculator, anyone cannot access SushiBeltTrackerItem useful properties.

### CODE REVIEW

**Review by jaxtynSong — APPROVED** [review]
👍

**Review by ElonPark — APPROVED** [review]
LGTM 👍👍

File: Sources/SushiBelt/SushiBeltTrackerItem.swift [hunk 1]
```diff
@@ -40,11 +40,11 @@ public struct SushiBeltTrackerItem {
     return self.isTracked ? .tracked : .tracking
   }
   
-  var rect: SushiBeltTrackerItemRect
-  var isTracked: Bool = false
-  var currentVisibleRatio: CGFloat = 0.0
-  var objectiveVisibleRatio: CGFloat = 0.0
-  var timestamp: Date
+  public var rect: SushiBeltTrackerItemRect
```
Comment (ppth0608) [inline Sources/SushiBelt/SushiBeltTrackerItem.swift:43]: 외부에서 write도 가능해질 것 같은데, 괜찮을까요?~
read only라면 `public private(set)`가 좋아보여요!

Comment (GeekTree0101) [reply Sources/SushiBelt/SushiBeltTrackerItem.swift:43]: you got it

**Review by ppth0608 — APPROVED** [review]
👍 👍

**Pull request merged**